client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -12,42 +13,75 @@ export const ASPECT_RATIO = 7 / 4;
 
 type PatternPreviewProps = {
 	isCategoryPreview?: boolean;
+	isResizable?: boolean;
 	pattern: Pattern | null;
 	viewportWidth?: number;
 };
 
-export function PatternPreview( {
+function PatternPreviewFragment({
 	isCategoryPreview,
 	pattern,
 	viewportWidth,
-}: PatternPreviewProps ) {
+}: PatternPreviewProps) {
 	const { renderedPatterns } = usePatternsRendererContext();
-	const patternId = encodePatternId( pattern?.ID ?? 0 );
-	const renderedPattern = renderedPatterns[ patternId ];
-	const [ resizeObserver, nodeSize ] = useResizeObserver();
+	const patternId = encodePatternId(pattern?.ID ?? 0);
+	const renderedPattern = renderedPatterns[patternId];
+	const [resizeObserver, nodeSize] = useResizeObserver();
```

Comment:
It looks like your ESLint/Prettier is acting up 🤔 Those extra spaces should be preserved

Reply:
Yes, it looks like I got the default formatter. I'll fix that.